pootle/apps/pootle_language/urls.py

```diff
@@ -7,14 +7,14 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
-from django.conf.urls import patterns, url
+from django.conf.urls import url
 
 from .views import (
     LanguageBrowseView, LanguageExportView, LanguageTranslateView)
+from . import views
```

Comment:
For consistency and to avoid a mixture, it'd be preferable to import from the `views` module when it's already done like that (check one line above). Could you please change this here and in other places where it applies?